pkg/model/openstackmodel/firewall.go

```diff
@@ -479,6 +480,13 @@ func (b *FirewallModelBuilder) Build(c *fi.ModelBuilderContext) error {
 			Name:      s(groupName),
 			Lifecycle: b.Lifecycle,
 		}
+		if role == kops.InstanceGroupRoleBastion {
+			sg.RemoveExtraRules = []string{"port=22"}
+		} else if role == kops.InstanceGroupRoleNode {
+			sg.RemoveExtraRules = []string{"port=22", "port=10250"}
```

Comment:
@drekle do you have idea why 10250 (kubelet port) is opened for all ips that are in kubernetesapiaccess? IMO that is not needed

Reply:
https://github.com/kubernetes/kops/blob/master/pkg/model/openstackmodel/firewall.go#L273-L295